HLshell/mycmd_1.py

```diff
@@ -82,17 +81,15 @@ def _checkInit(func):
         def wrapper(self, *params, **kargs):
             try:
                 if (self.fts is None):
-                    print_error('FTS is not yet initialized!')
-                    print_error('Did you mean to send the command instead?')
-                    return lambda *params, **kargs: None
-                elif (self.fts.state == FTSState.NOTINIT):
-                    print_err('FTS initialization failed!')
-                    return lambda *params, **kargs: None
```

Comment:
fts obj initialization may fail so that self.fts is not None but don't connect to XPS.

Reply:
I fixed it so that doesn't happen anymore